site/assets/js/search.js

```diff
@@ -5,51 +5,37 @@
 (() => {
   'use strict'
 
-  const inputElement = document.getElementById('search-input')
+  const siteDocsVersion = document.querySelector('[data-bd-docs-version]').getAttribute('data-bd-docs-version')
 
-  if (!window.docsearch || !inputElement) {
+  if (!window.docsearch) {
     return
   }
 
-  const siteDocsVersion = inputElement.getAttribute('data-bd-docs-version')
-
-  document.addEventListener('keydown', event => {
-    if ((((event.ctrlKey || event.metaKey) && event.key === 'k')) || (event.ctrlKey && event.key === '/')) {
-      event.preventDefault()
-      inputElement.focus()
-    }
-  })
-
-  if (navigator.platform.includes('Win') || navigator.platform.includes('Linux')) {
-    const searchShortcut = document.querySelector('.bd-search')
-    searchShortcut.setAttribute('data-shortcut', '⌃K')
-  }
-
   window.docsearch({
-    apiKey: '5990ad008512000bba2cf951ccf0332f',
```

Comment:
According to https://docsearch.algolia.com/docs/migrating-from-v2 `apiKey` must remain